contrib/html_demo/JupyterCode/20_upload_ui.ipynb

```diff
@@ -0,0 +1,243 @@
+{
```

Comment:
<p>Should add info here (or perhaps in readme) how to install azure since otherwise the from azure.storage.blob will throw an error (it's not part of the cv conda environment)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/528/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>